api/session/auth_session.py

```diff
@@ -35,6 +39,54 @@ def user():
                 error_response(404, "Session token expired or invalid")
         error_response(404, "No session token found")
 
+    @classmethod
+    def clear_session(cls):
```

Comment:
does this need to be a class method?

Reply:
You're right I don't think it does tbh - refactored to a staticmethod.